assignment/assignment.hs

```diff
@@ -41,7 +45,9 @@ toDigits num = "replace me"
 -- a given value occurs in a list
 
 count :: (Eq a, Num p) => a -> [a] -> p
-count element list = 0 -- replace me
+count element [] = 0
+count element (x:xs) = if (element == x) then (1 + count element xs) else (0 + count element xs)
+--i'm not sure if the parentheses are required but they make the code easier for me to read
 
```

Comment:
Yup!

Again, I tend to space things out a little:

~~~ hs
count val [] = 0 
count val (x:xs) =
    n + count val xs
    where 
        n = if val == x then 1 else 0
~~~

but that's just me... :)